app/controllers/trips_controller.rb

```diff
@@ -25,6 +26,19 @@ def destroy
     redirect_to '/trips'
   end
 
+  def edit
+    @trip = TripFacade.trip_by_id(@user.google_id, params[:id])
+  end
+
+  def update
+    # @trip = TripFacade.trip_by_id(@user.google_id, params[:id])
```

Comment:
We need to make sure to get rid of our many commented lines of code during refactor